practices/language.md

```diff
@@ -0,0 +1,57 @@
+# Language
+
+[Discrimination](https://en.wikipedia.org/wiki/Discrimination) is prohibited under law in the UK and many other countries.
+
+Language is our main form of communication and it plays a powerful role both in contributing to and in eliminating discrimination.
+
+Language influences thinking. For example, Hebrew uses gender markers, whereas Finnish doesn’t mark gender at all. A study done in the 1980s[1](http://onlinelibrary.wiley.com/doi/10.1111/j.1467-1770.1982.tb00973.x/abstract) found that kids who spoke Hebrew knew their own genders a year earlier than those who grew up speaking Finnish.
+
+Inclusive language is language that is free from words, phrases or tones that reflect prejudiced, stereotyped or discriminatory views of particular people or groups. It is also language that doesn't deliberately or inadvertently exclude people from being seen as part of a group.[2](https://www.education.tas.gov.au/.../Documents/Guidelines-for-Inclusive-Language.pdf)
+
+We are commited to use inclusive, non-discriminatory language in every written or verbal communication.
+
+
+## Unacceptable language
+
+Inappropriate or unacceptable language highlights perceived or actual differences between people. The use of insensitive and inappropriate language can create a hostile environment for people who have protected characteristics, as outlined in the [Equality Act 2010](http://www.legislation.gov.uk/ukpga/2010/15/contents).
+
+Stereotyping means presuming a range of things about people based on one or two of their personal characteristics such as their appearance, apparent intelligence, personality or character, or their gender, sexual orientation, race, ethnicity, age, location, socioeconomic status or disability. We don't use language that reinforces stereotypes.
+
+
+## Examples
+
+
+### Genderless language
+
+* When writing documentation or examples that don't refer to specific people, we prefer the gender neutral honorific [`Mx`](http://www.oxforddictionaries.com/definition/english/mx#nav2).
```

Comment:
R.e. Mx: This is wonderful. I don't think it should be in our documentation, though :D 

- It's a self-selected designation, and definitely not something that should be assigned to an individual without their permission (no different to referring to a woman as "Mr" or to a man as "Miss")
- It's a title, and honestly there isn't much of a use-case for titles in technical documentation unless it's documenting how to directly address users